client/client.go

```diff
@@ -104,6 +104,9 @@ type Config struct {
 	//
 	// A HeaderTimeoutPerRequest of zero means no timeout.
 	HeaderTimeoutPerRequest time.Duration
+
+	// send requests to leader in default for avoiding forwarding from follower to leader
+	PrioritizeLeader bool
```

Comment:
How about changing this to Preference? If preference is Leader, then we try our best to send request to leader. In the future, we can prefer a member name or even ip range.